From 2631ef52e82ef2622f45f56d13cb882f05dfc34b Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 1 Sep 2015 09:58:02 +0800
Subject: [PATCH 0646/5242] MLK-11465-2: ARM: dts: imx7d/imx6ul: update SW1A
 for PFUZE3000

commit  852cec1a7a9b42bca61404a1f327e6414a3b4fce from
https://source.codeaurora.org/external/imx/linux-imx.git

Update max voltage of SW1A to 3.3V on PF3000, see below datasheet:
http://cache.freescale.com/files/analog/doc/data_sheet/PF3000.pdf?fsrch=1&sr=1&pageNum=1

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit 5cd7d3c5933b052b899183a45fa4be8cdb2a0c69)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-9x9-evk.dts          |    8 ++++++++
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    2 +-
 arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts |    2 +-
 arch/arm/boot/dts/imx7d-sdb.dts               |    2 +-
 4 files changed, 11 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-9x9-evk.dts b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
index 901dbfd..08088cf 100644
--- a/arch/arm/boot/dts/imx6ul-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
@@ -236,6 +236,14 @@
 		reg = <0x08>;
 
 		regulators {
+			sw1a_reg: sw1a {
+					regulator-min-microvolt = <700000>;
+					regulator-max-microvolt = <3300000>;
+					regulator-boot-on;
+					regulator-always-on;
+					regulator-ramp-delay = <6250>;
+			};
+
 			/* use sw1c_reg to align with pfuze100/pfuze200 */
 			sw1c_reg: sw1b {
 				regulator-min-microvolt = <700000>;
diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index daad2b2..fbc6839 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -243,7 +243,7 @@
 		regulators {
 			sw1a_reg: sw1a {
 				regulator-min-microvolt = <700000>;
-				regulator-max-microvolt = <1475000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 				regulator-always-on;
 				regulator-ramp-delay = <6250>;
diff --git a/arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts b/arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts
index fffaca2..5573afb 100644
--- a/arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts
@@ -103,7 +103,7 @@
 		regulators {
 			sw1a_reg: sw1a {
 				regulator-min-microvolt = <700000>;
-				regulator-max-microvolt = <1475000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 				regulator-always-on;
 				regulator-ramp-delay = <6250>;
diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 63d0598..b813e37 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -352,7 +352,7 @@
 		regulators {
 			sw1a_reg: sw1a {
 				regulator-min-microvolt = <700000>;
-				regulator-max-microvolt = <1475000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 				regulator-always-on;
 				regulator-ramp-delay = <6250>;
-- 
1.7.9.5

